indra/newview/llmodelpreview.cpp

```diff
@@ -272,6 +272,10 @@ void LLModelPreview::rebuildUploadData()
 
     mUploadData.clear();
     mTextureSet.clear();
+    if (mDefaultPhysicsShapeP && mDefaultPhysicsShapeP->getNumRefs() == LL_REFCOUNT_FREE)
```

Comment:
Why does it end in this state?

Reply:
This was a root cause of the bug
The object was deleted but mDefaultPhysicsShapeP still had it's address and then it was reused
mDefaultPhysicsShapeP  is not a LLPointer and this is right - it shouldn't prevent deleting the object

Reply:
>mDefaultPhysicsShapeP is not a LLPointer and this is right - it shouldn't prevent deleting the object

It can be "if (mDefaultPhysicsShapeP->getNumRefs() == 1) mDefaultPhysicsShapeP = nullptr;" or in case of a weak pointer, check if it is alive. The problem is you are calling getNumRefs() from a dead object

Reply:
> The problem is you are calling getNumRefs() from a dead object

Agree. We may null this pointer unconditionally. Fixed. Thanks!